ui/src/pages/gcp/login.js

```diff
@@ -0,0 +1,338 @@
+            window.location.reload();
+            return <div />;
+        }
+        if (this.props.error) {
+            return <Error err={this.props.error} />;
+        }
+        if (this.state.isFetching) {
+            return <ReduxPageLoader message={'Loading resource access list'} />;
+        }
+        let displayProjects = [];
+        for (let pName in this.state.projectRoleMap) {
+            let projectRoleNames = [];
+            this.state.projectRoleMap[pName].forEach((projectObject, index) => {
+                projectRoleNames.push(
+                    <div key={index}>
+                        <RadioButton
+                            type={'radio'}
+                            key={'button-' + projectObject.projectId}
+                            value={projectObject.projectId}
+                            name={'project'}
+                            checked={
+                                this.state.projectRole ===
+                                projectObject.projectId
+                            }
+                            onChange={() =>
+                                this.handleRadioButton(projectObject)
+                            }
+                            required={true}
+                        />
+                        <ProjectLabel key={'label-' + projectObject.projectId}>
+                            {projectObject.projectRoleName}
+                        </ProjectLabel>
+                    </div>
+                );
+            });
+
+            displayProjects.push(
+                <div>
+                    <ProjectTitleDiv key={'project-name-' + pName}>
+                        Project: {pName}
+                    </ProjectTitleDiv>
+                    <RadioButtonsContainer
+                        key={'radio-button-container-' + pName}
+                    >
+                        {projectRoleNames}
+                    </RadioButtonsContainer>
+                </div>
+            );
+        }
+
+        if (!displayProjects.length) {
+            return (
+                <CacheProvider value={this.cache}>
+                    <div data-testid='gcp-login-error'>
+                        <GcpHeader>
+                            <img src='/static/google-cloud.svg'></img>
+                        </GcpHeader>
+                        <ParentWrapperDiv>
+                            <h3>
+                                Error: There are no GCP project roles associated
+                                with your account.
+                            </h3>
+                            <p>
+                                Check to make sure that your <tt>gcp.*</tt>{' '}
+                                roles contain your user.
+                                <br />
+                                E.g., if your account is 'jdoe', then{' '}
+                                <tt>gcp.fed.admin.user</tt> should have{' '}
+                                <tt>user.jdoe</tt> as a member.
```

Comment:
Instead of `user.jdoe`, can you use `consts.USER_DOMAIN + ".jdoe"`

https://github.com/AthenZ/athenz/blob/f7791d91ed7709110d7644a5cd9ed28cd7409c64/ui/src/components/constants/constants.js#L25

Reply:
fixed.